src/Microsoft.Health.Dicom.Core/Features/Common/DicomValuelessItem.cs

```diff
@@ -0,0 +1,27 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Linq;
+using FellowOakDicom;
+
+namespace Microsoft.Health.Dicom.Core.Features.Common
+{
+    /// <summary>
+    /// A DicomItem with a tag but no value - used to model either concrete DicomElements or DicomSequences.
+    /// </summary>
+    public class DicomValuelessItem : DicomItem
```

Comment:
What is DicomValueLessItem adding to DicomItem? Do we need this?

Reply:
Yes, we need this, Dicom Sequence is a DICOM item and it is abstract. unlike our core tag which can be represented as DICOM tag. We need a way to store it in an object model from the DB.

Reply:
Removing this file